xtask/src/commands/test.rs

```diff
@@ -28,6 +29,23 @@ impl Test {
             "--no-demo and --with-demo are mutually exclusive",
         );
 
+        // start building tests, while the subservices are spinning up
+        eprintln!("Starting to build tests...");
+        let _build = std::process::Command::new(which::which("cargo")?)
+            .args([
+                "test",
```

Comment:
Use `cargo build --tests` instead otherwise you will be running the tests

Reply:
the "--no-run" flag allows me to build the tests and not run them https://doc.rust-lang.org/cargo/commands/cargo-test.html#test-options

Reply:
Ohh I didn't know that one!

`cargo build --tests` does exactly the same I guess then!

Reply:
both are really close indeed! given I try to reuse the build cache as much as possible, I'm looking for the closest argument list to pass to cargo, and try not to confuse it 😁 !

Reply:
`build --tests` is definitely clearer imo :trollface: